src/exometer_report_influxdb.erl

```diff
@@ -136,6 +137,34 @@ exometer_report(Metric, DataPoint, _Extra, Value,
             Error
     end.
 
+exometer_report_bulk(_Found, _Extra, #state{connection = undefined} = State) ->
+    ?info("InfluxDB reporter isn't connected and will reconnect."),
+    {ok, State};
+exometer_report_bulk([{_Metric,[{_DataPoint, _Value}|_]}|_] = Found, _Extra, 
+                     #state{metrics = Metrics} = State) ->
+    {NState, Errors} = 
+        lists:foldl(
+          fun({Metric, DataPoints}, {StateAccIn, ErrorsAccIn}) -> 
+                  case maps:get(Metric, Metrics, not_found) of
+                      {MetricName, Tags} ->
+                          case maybe_send(Metric, MetricName, Tags, 
```

Comment:
Did you notice that `maybe_send` has two behaviors depends on `batch_window_size`?
With `batch_window_size=0` each metric from a bulk will be sent in an own request.
With `batch_window_size>0` it isn't sent immediately but goes to a buffer to the further sending in one batch.